dokka-subprojects/plugin-base/src/main/resources/dokka/styles/style.css

```diff
@@ -336,33 +336,23 @@ code.paragraph {
     padding: 0;
 }
 
-.symbol:not(.token), code {
-    background-color: var(--code-background);
-    align-items: center;
-    box-sizing: border-box;
-    white-space: pre-wrap;
-    font-family: var(--default-monospace-font-family);
-    font-size: var(--default-font-size);
-}
-
 .symbol:not(.token), code.block {
```

Comment:
What do you thing about moving this part to /inline-code in the ui-kit to keep all `<code/>` styles closer?

Reply:
I want to create a separate "block code" component since it's more complicated than an inline one.
Will be done in [KTL-1707](https://youtrack.jetbrains.com/issue/KTL-1707) Dokka: adjust code area to new design